### DISCUSSION TITLE
How to save keybindings on "Miscellaneous" GUI page?

### DISCUSSION THREAD

**Discussion by ghost** [BlitterStudio/amiberry#1469] [Q&A] [answered]
I wanted to set a key combo for "Minimize" but there seems to be no way to save it. Saving active configuration does not save this keybinding. I've also tried to change "Open GUI" to some other key, but it is not saved as well.

**Comment by giantclambake** [comment, marked as answer]
These settings are saved in the currently loaded config.uae file.

To apply these 'globally' so to speak, you have to manually edit your amiberry.conf file ....example...

```
default_open_gui_key=F12
default_quit_key=Keypad *
default_ar_key=Pause
default_fullscreen_toggle_key=Keypad /
```
Once you save those edits and start amiberry, the keypress mappings will apply....that said...

@midwan ~ there is *no* entry in amiberry.conf  for ie; `default_minimize_key=`  ...is that correct, or an oversight?

**Reply by midwan** [reply]
There never was a need to have a default minimize key, so we never created one. :)
